docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+    | name                         | nanisivik_vol                                                     |
+    | os-vol-tenant-attr:tenant_id | 94109c764a754e24ac0f6b01aef82359                                  |
+    | properties                   |                                                                   |
+    | replication_status           | None                                                              |
+    | size                         | 56                                                                |
+    | snapshot_id                  | None                                                              |
+    | source_volid                 | None                                                              |
+    | status                       | available                                                         |
+    | type                         | ceph_hdd                                                          |
+    | updated_at                   | 2023-05-10T13:39:05.000000                                        |
+    | user_id                      | c096cf99f65a4d22a6954b67d2ec11d7                                  |
+    | volume_image_metadata        | {'hw_qemu_guest_agent': 'yes', 'base_image_ref':                  |
+    |                              | 'd6b012ee-c6d3-4672-9399-b87d025ddb14', 'owner_user_name':        |
+    |                              | 'kolderson', 'owner_project_name': 'Meanwhile-in-Drama',          |
+    |                              | 'boot_roles': 'load-                                              |
+    |                              | balancer_member,_member_,swiftoperator,creator',                  |
+    |                              | 'hw_machine_type': 'pc', 'instance_uuid':                         |
+    |                              | '19ffeb4d-3016-481f-886e-6caecc525fc2', 'user_id':                |
+    |                              | 'c096cf99f65a4d22a6954b67d2ec11d7', 'image_type': 'snapshot',     |
+    |                              | 'owner_id': '94109c764a754e24ac0f6b01aef82359', 'image_state':    |
+    |                              | 'available', 'image_location': 'snapshot', 'image_id':            |
+    |                              | '1a50720b-af1c-4a16-ba19-b89dbee251f0', 'image_name':             |
+    |                              | 'nanisivik_snap', 'container_format': 'bare', 'disk_format':      |
+    |                              | 'raw', 'min_disk': '50', 'min_ram': '0', 'size': '53687091200'}   |
+    +------------------------------+-------------------------------------------------------------------+
+    ```
+
+## Deleting the boot-from-image server
+
+=== "{{gui}}"
+    Go to the servers view by selecting Compute →
+    [Servers](https://{{gui_domain}}/compute/servers), and locate the
+    boot-from-image server. It is time to delete it, so click the orange
+    :fontawesome-ellipsis-circle: icon at the right of the server row, and
```

Comment:
Again, `:fontawesome-ellipsis-circle:` is not one of [shortcodes](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/) that mkdocs-material supports.